keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -184,6 +210,15 @@ class SqlDeliveryConfigRepository(
         .fetch(ENVIRONMENT_RESOURCE.RESOURCE_UID)
     }
 
+  private fun getResourceIDs(application: String): List<String> =
+    sqlRetry.withRetry(READ) {
+      jooq
+        .select(RESOURCE.ID)
+        .from(RESOURCE)
+        .where(RESOURCE.APPLICATION.eq(application))
+        .fetch(RESOURCE.ID)
+    }
```

Comment:
Seems to me it would be pretty easy to inline this into the queries and delete everything at once rather than having to do a `forEach` over all the resources

Reply:
Ya, good point. I just got lazy and followed the pattern. Will change it!